app/components/annotation/Answer.js

```diff
@@ -1,3 +1,4 @@
+// TODO: marked for deletion: topic picking view doesn't need Q&A
```

Comment:
as i understand it the annotation view (topicPicker, article) won't display questions that the user has to answer as it might have been planned in a previous UI design. instead those questions will just be asked in the quiz view, so we won't need these `Question` and `Answer` components - they'll get reborn in the `QuizQuestion` and `QuizAnswer` components